### DISCUSSION TITLE
Prisma ORM docs as EPUB

### DISCUSSION THREAD

**Discussion by mephju** [prisma/prisma#24788] [Q&A]
### Question

Hello good people.

Do you know if it is possible to download the Prisma docs as EPUB? I would love to read through the docs on my ebook reader.

#

**Comment by ludralph** [comment]
Hi @mephju 

Thank you for raising this question.

Currently we do not support downloading the documentation as an EPUB. You can however open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so our engineers can take a look. As a workaround, you can try this third-party service [dotepub](https://dotepub.com/converter/?lang=en).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
